docs/usecases/RStudio_user.rst

```diff
@@ -0,0 +1,221 @@
+.. \_usecase_Rstat:
+
+My first steps as a RStudio users
+---------------------------
+
+.. index:: ! Usecase; R users quickstart
+
+This use case sketches typical entry points for R and Rstudio users. 
+
+#. A repository having submodules for data and code is cloned.
+#. R scripts are developped in Rstudio
+#. R scripts are run usind datalad run
+
+This is a `hello world` type of analysis, used only for demonstration purposes
+
+The Challenge
+^^^^^^^^^^^^^
+
+Max has been using Rstudio together with GitHub for a long time. They know how git
+works. Max has learned that git will not work with their new project,
+because there will be too many files and some dataset files will be too large.
+Max read the datalad handbook basics and is decided to apply datalad.
+Max still want to use Rstudio and a combination of R and python scripts for the
+data analysis.
+
+Bobby is a data manager who already learned (the hard way), how to handle datalad
+using Rstudio. They have also created a GIN repository with submodules 
+for data and for code, using the Tonic tool and templates.
+
+
+Setting up
+^^^^^^^^^^
+
+Max first want to clone the repository on their computer, they use the Rstudio 
+`create a new project` function using the SSH address of the parent repository.
+Max can't see submodules content and come to Bobby.
+
+Bobby comes and run `datalad get . -n -r` in the terminal window of Rstudio. 
+
+They then explain:
+  - Rstudio can only use simple git commands, which do not clone submodule content.
+  - datalad command are run in the terminal window. Datalad does not have a R package and do not run in the console
+  - This specific function `get .` will download all files, it has two options:
+     - `-n` option means annexed files will not be downloaded
+     - `-r` option means that the function is run in all submodules, recursively
+
+Max has learned something, and it realise that git-annex is probably not set in its repositoy.
+After reading the handbook, they see the create function need to be forced 
+when the folder already exist, so they  run 
+`datalad create --force -r` in the parent repository.
+Now they are sure they set up datalad to work in the repository and all submodules,
+since they used the `-r` option.
+
```

Comment:
I feel like this can conclude with a statement that with this setup, everything is good to go for DataLad commands from the console, for example for saving changes,pushing modifications, pulling updates, or adding siblings.

Reply:
Added some infos in the box, indeed pushing would need additional setups in the scenario